src/app/lib/utilities/getUrlHelpers/getEmbedUrl/index.js

```diff
@@ -28,27 +24,26 @@ const shouldOverrideMorphEnv = (queryString, type) => {
 
 const isDev = () => getEnvConfig().SIMORGH_APP_ENV === 'local';
 
-const getBaseUrl = (queryString, isAmp) => {
+const getBaseUrl = () => {
   // In some scenarios, we use the same base URL as the parent
   const relativeBaseUrl = '';
   switch (true) {
     case isLive():
-      return isAmp ? LIVE_AMP_URL : relativeBaseUrl;
+      return relativeBaseUrl;
     case isDev():
-      return isAmp ? DEV_AMP_URL : DEV_BASE_URL;
+      return DEV_BASE_URL;
     default:
-      return isAmp ? TEST_AMP_URL : relativeBaseUrl;
+      return relativeBaseUrl;
   }
 };
```

Comment:
Could I suggest a wee refactor / simplication while we're in here?

So you'll notice the logic is now:

if dev, return the dev base URL otherwise return empty string

Could you try to implement ☝️ as code?

Could I also suggest that we use `isLocal()` from https://github.com/bbc/simorgh/blob/1259babbd823b479491223795ae0cb9020ac989e/src/app/lib/utilities/isLocal/index.js and delete `isDev()`?

And perhaps update all references to `DEV` to `LOCAL`

The tests should keep you right 👍

Reply:
Marking as resolved due to the changes in this file being reverted.